common/opinion_modifiers/wc_crime_and_prison_opinions.txt

```diff
@@ -0,0 +1,11 @@
+﻿stasis_traped_me_opinion = {
+	opinion = -30
+	years = 20
+	decaying = yes
+}
+
+attempted_stasis_trapion_opinion = {
+	opinion = -20
+	years = 10
+	decaying = yes
+}
```

Comment:
It's still not a crime.
One player can do it against another player and not having a reason to punish.